backend/app/api/endpoints/admin/im_channels.py

```
@@ -0,0 +1,412 @@
+        created_at=channel.created_at,
+        updated_at=channel.updated_at,
+        created_by=channel.create_user_id,
+    )
+
+
+@router.get("/im-channels", response_model=IMChannelListResponse)
+async def list_im_channels(
+    page: int = Query(1, ge=1),
+    limit: int = Query(20, ge=1, le=100),
+    channel_type: Optional[str] = Query(None, description="Filter by channel type"),
+    db: Session = Depends(get_db),
```

Ruff (0.14.14)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)